openapi/openapi.yaml

```diff
@@ -972,6 +972,7 @@ paths:
       parameters:
       - $ref: "#/components/parameters/page"
       - $ref: "#/components/parameters/per_page"
+      - $ref: "#/components/parameters/order_by"
```

Comment:
Please add parameter description in the component defination
https://github.com/thoth-station/user-api/blob/b80e0d55faa9a88c67119ee235a81dd28e9954e9/openapi/openapi.yaml#L1104

Reply:
added description